superchain/extra/README.MD

```diff
@@ -1,4 +1,8 @@
 # extra
 
-This is a collection of extra configuration,
-since not all address and system-config information is available through L1 for some chains.
+This is a collection of extra configuration data because not all address and system-config information is available through L1 for some chains.
+
+- [addresses](./addresses/): L1 Smart Contract addresses on for each network, grouped by Superchain target.
```

Comment:
This information now lives in the toml config files so this dir should not be referenced